packages/js/src/generators/library/schema.json

```diff
@@ -18,23 +18,27 @@
     },
     "directory": {
       "type": "string",
-      "description": "A directory where the lib is placed."
+      "description": "A directory where the lib is placed.",
+      "x-priority": "important"
     },
     "linter": {
       "description": "The tool to use for running lint checks.",
       "type": "string",
       "enum": ["eslint", "none"],
-      "default": "eslint"
+      "default": "eslint",
+      "x-priority": "important"
     },
     "unitTestRunner": {
       "type": "string",
       "enum": ["jest", "vitest", "none"],
       "description": "Test runner to use for unit tests.",
-      "x-prompt": "Which unit test runner would you like to use?"
+      "x-prompt": "Which unit test runner would you like to use?",
+      "x-priority": "important"
     },
     "tags": {
       "type": "string",
-      "description": "Add tags to the library (used for linting)."
+      "description": "Add tags to the library (used for linting).",
+      "x-priority": "important"
     },
     "skipFormat": {
```

Comment:
Are`skipFormat` and `skipTsConfig` really used by actual people? Because those are good candidates for `internal` properties imo

Reply:
Agree. I also added `internal` to a couple more properties